install/kubernetes/helm/subcharts/tracing/values.yaml

```diff
@@ -66,3 +66,6 @@ ingress:
     #   hosts:
     #     - tracing.local
 gatewayName: ingressgateway
```

Comment:
Would it be possible to move this under the gatrway node, so:

```
gateway:
  enabled: false
  name: ingressgateway
```
I

Reply:
Yup - done. See commit 822c690a51dc3eb1afdd7ac61eedaa162dcbfca2